bbc/simorgh
# WSTEAM1-758: CAF updates for STY/ Article Pages

Resolves JIRA [WSTEAM1-758](https://jira.dev.bbc.co.uk/browse/WSTEAM1-758)

Overall changes
======
Makes required updates to CAF rendered STY/ Article pages so that they are ready to release using a low risk approach.

Code changes
======

- Show related content images and add unit test
- Add SEO description field to article page schema
- No change to Tracking events values since they are consistent with existing STY pages. (See write up in a comment on the subtask: https://jira.dev.bbc.co.uk/browse/WSTEAM1-921)

Testing
======
Related Content
1. Pull down this PR and run Simorgh locally
2. Visit this link: http://localhost:7080/japanese/67305510?renderer_env=caflive
3. Check that all Related Content Item images render

SEO description

1. Pull down this PR and run Simorgh locally
2. Visit this link: http://localhost:7080/mundo/23263889?renderer_env=caftest
3. Use Detailed SEO Extension, go to the Schema tab and check the 'description' value is present. 
Also see comments on this subtask: https://jira.dev.bbc.co.uk/browse/WSTEAM1-921

Tracking
1. Follow instructions/ see screenshots in the comments on this subtask: https://jira.dev.bbc.co.uk/browse/WSTEAM1-921)

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
